ci/ansible.cfg

```diff
@@ -1,5 +1,5 @@
 [defaults]
-callbacks_enabled = custom_logger
+callbacks_enabled = custom_logger, junit
```

Comment:
This is not required for this change
```suggestion
callbacks_enabled = custom_logger
```

Reply:
These commits are related to https://github.com/infrawatch/feature-verification-tests/pull/136

Reply:
They don't need to be included in this PR though. 
The only changes on this branch should be related to updating the autoscaling tests.